src/test/Assets/TestProjects/StandaloneApp20/StandaloneApp20.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<AssemblyName>StandaloneApp</AssemblyName>
```

Comment:
Why not keep the assembly name the same as the project?  I think that would avoid confusion later, wouldn't it?

Reply:
Because the old and new versions of the apphost are swapped depending on the test, and we need to have the sane embedded apphost.name